marketplace/deployer_util/constants.py

```diff
@@ -13,3 +13,5 @@
 # limitations under the License.
 
 GOOGLE_CLOUD_TEST = 'marketplace.cloud.google.com/verification'
+
+LOG_SMOKE_TEST = 'LOG_SMOKE_TEST'
```

Comment:
I wonder if this is providing good enough value to justify over the inconsistency with what we currently have in our code. It seems that it's clear enough that these messages are talking about the tests.

My recommendation is to remove these log prefixes, in both sh and py.

Reply:
I am adding them to help parse out for the report. Think of them as something that will be consumed by our code down the road. It will help have them label so we can control what to show and where.